Neos.Flow/Classes/Property/TypeConverter/PersistentObjectConverter.php

```diff
@@ -215,6 +221,9 @@ public function convertFrom($source, $targetType, array $convertedChildPropertie
                 );
                 throw new InvalidTargetException($exceptionMessage, 1297935345);
             }
+            if ($isCollectionType && $originalValue !== ObjectAccess::getProperty($object, $propertyName, true)) {
+                throw new TypeConverterException(sprintf('You overwrote the collection property "%s" of type "%s". This will lead to data loss. Read more about it and how to prevent this here: ADDLINKHERE', $propertyName, $targetType), 1602324196);
```

Comment:
```suggestion
                throw new TypeConverterException(sprintf('You overwrote the collection property "%s" of type "%s". This will lead to data loss. Read more about it and how to prevent this here: https://www.neos.io/go/flow-7-0-release-notes.html', $propertyName, $targetType), 1602324196);
```

Reply:
^ This "short link" will redirect to https://flowframework.readthedocs.io/en/7.0/TheDefinitiveGuide/PartV/ReleaseNotes/index.html once the release is done